controller/src/main/java/cern/jarrace/controller/Application.java

```diff
@@ -0,0 +1,25 @@
+package cern.jarrace.controller;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+/**
+ * Main entry point for the controller, instanciates an embedded Jetty container server, loads the spring
+ * {@link org.springframework.web.servlet.DispatcherServlet} and configures all the
+ * {@link org.springframework.web.bind.annotation.RestController}s
+ * @author timartin
+ */
+
+@SpringBootApplication
+public class Application {
```

Comment:
What also comes into my mind: Could you also make an example on how to integrate such a server in another (existing) server?